database/database.go

```diff
@@ -322,18 +322,28 @@ func (db *Database) SetLayerCached(digest string, cached bool) (err error) {
 
 // AddInstance adds instance information to db.
 func (db *Database) AddInstance(instance launcher.InstanceInfo) error {
-	return db.executeQuery("INSERT INTO instances values(?, ?, ?, ?, ?, ?, ?, ?)",
+	network, err := json.Marshal(instance.NetworkParameters)
+	if err != nil {
+		return aoserrors.Wrap(err)
+	}
+
+	return db.executeQuery("INSERT INTO instances values(?, ?, ?, ?, ?, ?, ?, ?, ?)",
 		instance.InstanceID, instance.ServiceID, instance.SubjectID, instance.Instance, instance.UID,
-		instance.Priority, instance.StoragePath, instance.StatePath)
+		instance.Priority, instance.StoragePath, instance.StatePath, network)
```

Comment:
What about covering it with unit test?

Reply:
Yes, you are correct I forgot to add them.